src/NuGet.Services.V3/DependencyInjectionExtensions.cs

```diff
@@ -50,5 +60,60 @@ public static IServiceCollection AddV3(this IServiceCollection services, IDictio
 
             return services;
         }
+
+        public static void AddFeatureFlags(this ContainerBuilder containerBuilder)
+        {
+            containerBuilder
+                .Register(c =>
+                {
+                    var options = c.Resolve<IOptionsSnapshot<FeatureFlagConfiguration>>();
+                    var requestOptions = new BlobRequestOptions
+                    {
+                        ServerTimeout = TimeSpan.FromMinutes(2),
```

Comment:
Should we make it a private const value to configure the timeout? Maybe it's fine here because we only use the value at this place.

Reply:
Added